CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Version 5.0.0 is a backwards-incompatible release. Please see the [upgrade instr
 - Cleaning up unused input variables. [#300]
 - Add ability to set net_write_timeout for CloudSQL [#299]
 - Update server config template to reflect CSCC notifier changes [#292]
+- Adding manage_server_config_enabled variable [#326]
```

Comment:
There is another bucket with states that should be preserved, and that is the `forseti-client-xxxxxx` bucket.

Reply:
I didn't add this to the client bucket for the following reason.  Each time the server VM is destroyed and recreated, the server vm gets a new internal IP.  Therefore the config in the client bucket needs to be updated.